tools/afl/generate/smessage_encrypt_decrypt.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019 Cossack Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/cossacklabs/themis/gothemis/keys"
+	"github.com/cossacklabs/themis/gothemis/message"
+)
+
+func writeByteString(w io.Writer, str []byte) {
+	binary.Write(w, binary.BigEndian, uint32(len(str)))
+	w.Write(str)
+}
+
+func main() {
+	args := os.Args
+
+	if len(args) != 3 {
+		fmt.Printf("usage:\n\t%s {EC|RSA} <message>\n", args[0])
+		os.Exit(1)
+	}
+
+	keyType := -1
+	switch args[1] {
+	case "ec", "EC":
+		keyType = keys.TypeEC
+	case "rsa", "RSA":
+		keyType = keys.TypeRSA
+	default:
+		fmt.Fprintf(os.Stderr, "unknown key type: %s\nsupported: EC, RSA\n", args[1])
+		os.Exit(1)
+	}
+	keypair, _ := keys.New(keyType)
```

Comment:
why we skip error checks here and below?

Reply:
> why we skip error checks here and below?

Because I was lazy 😁 This tool is intended for Themis maintainers, not users, so proper error handling is not really a must since this thing shouldn't fail in the first place. But okay, I'll add some failure reporting here, just in case.

> what if we miss to install themis to the system?

In that case the tool will fail to compile, as CGo wouldn't be able to find Themis Core.